src/settings.scad

```diff
@@ -92,7 +92,7 @@ $dish_overdraw_height = 0;
 $cherry_bevel = true;
 
 // How tall in mm the stem support is, if there is any. stem support sits around the keystem and helps to secure it while printing.
-$stem_support_height = 0.4;
+$stem_support_height = .8;
```

Comment:
this adversely affects brim a bit, since it doesn't need to be that high, though I did rip off a few brims today. this is mostly for a complementing change in tines.scad, where the tines are thinner but taller. it makes more sense, since most of the force will be torque from the print head crashing into the curling corners of the stem support